lib/components/inputs/inputview.coffee

```diff
@@ -449,14 +451,18 @@ module.exports = class KDInputView extends KDView
 
     # input content is copied into clone
     # element to get calculated height
-    @_clone = $ '<div/>', class : 'invisible'
+    @_clone = $ '<div/>', { class: 'invisible' }
+
+    { type } = @getOptions()
+    isVertical = type.toLowerCase() is 'textarea'
```

Comment:
Assumption we did earlier was to use autogrow only for textareas and just vertically, now we expand the functionality to `text` inputs basically, but they can only grow horizontally. That's why I didn't think another option was necessary.

However I didn't think about a bidirectional growing input, never needed that. Do you think it is necessary?